puppet/modules/quickstack/manifests/dellsc/volume.pp

```diff
@@ -1,4 +1,4 @@
-define quickstack::dell_sc_iscsi::volume (
+define quickstack::dellsc::volume (
```

Comment:
Why are we dropping iscsi?
I think we just need to have consistent name. dell_sc_iscsi does make sense to me. We use that for documentation for kilo.